cartographer/pose_graph/constraint/constraint_utils.h

```diff
@@ -25,6 +25,21 @@
 namespace cartographer {
 namespace pose_graph {
 
+#define FIND_NODE(node_name, node_id, map, log_message)                   \
+  FIND_NODE_IMPL(STATUS_MACROS_IMPL_CONCAT_(__node, __LINE__), node_name, \
+                 node_id, map, log_message)
+
+#define FIND_NODE_IMPL(node_item, node_name, node_id, map, log_message) \
+  auto node_item = common::FindOrNull(map, node_id);                    \
```

Comment:
why not direclty?:
auto node_name = common::FindOrNull(map, node_id);

Then you don't need these concatenations and extra macros